plugins/setting/src/index.ts

```diff
@@ -76,6 +76,15 @@ export interface SettingsCategory extends Doc {
   secured: boolean
 }
 
+/**
+ * @public
+ */
+export interface InviteSettings extends Doc {
```

Comment:
Please extend Configuration base class, this kind of settings should be available only for owners of workspace.

Reply:
Ok, let's do in next PR.